src/components/Tooltip/docs/testProps.ts

```diff
@@ -0,0 +1,10 @@
+import { TooltipSkin } from '../TooltipEnums';
+
+export const testPropsList: Partial<any | { ['data-hook']: string }>[] = [
```

Comment:
`Partial<any | { ['data-hook']: string }>[]` resolves to `Partial<any>[]` which can be expressed as `{[key:string]: any}[]`

Reply:
Initially I was using it like `Partial<TooltipProps | { ['data-hook']: string }>[]`. But I faced with the problem that I can't separate  `TooltipProps` and `jsx` syntax. `jsx` syntax is not supported in `e2e` so I put it as `any`.

Reply:
I'll change it as you are telling me.